docs/framework/net-native/example-handling-exceptions-when-binding-data.md

```diff
@@ -5,7 +5,7 @@ ms.assetid: bd63ed96-9853-46dc-ade5-7bd1b0f39110
 ---
 # Example: Handling Exceptions When Binding Data
 > [!NOTE]
-> This topic refers to the .NET Native Developer Preview, which is pre-release software. You can download the preview from the [Microsoft Connect website](https://go.microsoft.com/fwlink/?LinkId=394611) (requires registration).  
+> This topic refers to the .NET Native Developer Preview, which is pre-release software. You can download the preview from the [Microsoft Connect website](https://docs.microsoft.com/dotnet/framework/net-native/index) (requires registration).  
```

Comment:
undo this change. Connect is gone and I'm trying to get the right answer here on what to do with this content.

```suggestion
> This topic refers to the .NET Native Developer Preview, which is pre-release software. You can download the preview from the [Microsoft Connect website](https://go.microsoft.com/fwlink/?LinkId=394611) (requires registration).  
```